examples/regression/1-quickstart/plot_prefit.py

```diff
@@ -3,21 +3,26 @@
 Example use of the prefit parameter with neural networks
 ========================================================
 
-:class:`mapie.regression.MapieRegressor` is used to calibrate
-uncertainties for large models for which the cost of cross-validation
-is too high. Typically, neural networks rely on a single validation set.
+:class:`mapie.regression.MapieRegressor` and
+:class:`mapie.quantile_regression.MapieQuantileRegressor``
+are used to calibrate uncertainties for large models for
+which the cost of cross-validation is too high. Typically,
+neural networks rely on a single validation set.
 
 In this example, we first fit a neural network on the training set. We
 then compute residuals on a validation set with the `cv="prefit"` parameter.
 Finally, we evaluate the model with prediction intervals on a testing set.
 """
-import scipy
+
+
 import numpy as np
-from sklearn.model_selection import train_test_split
-from sklearn.neural_network import MLPRegressor
+from lightgbm import LGBMRegressor
```

Comment:
The use of `LGBMRegressor` is not compatible with the title of this example ;-). Besides, e wanted to use a MLP to emphasize the fact that MAPIE can be used with deep learning models. Use LGBM as a second model in addition to MLPs and change the title ?

Reply:
changed